virttest/utils_v2v.py

```diff
@@ -182,6 +205,7 @@ def get_cmd_options(self, params):
         self.input_file = self.params.get('input_file')
         self.new_name = self.params.get('new_name')
         self.username = self.params.get('username', 'root')
+        self.vpx_no_username = self.params.get('vpx_no_username')
```

Comment:
Params_get can read the variants in cfg file directly. 
The old code style is not good and looks bad.